sql/core/src/test/scala/org/apache/spark/sql/DatasetAggregatorSuite.scala

```diff
@@ -84,6 +84,16 @@ object ComplexBufferAgg extends Aggregator[AggData, (Int, AggData), Int] {
     (b1._1 + b2._1, b1._2)
 }
 
+object nameAgg extends Aggregator[AggData, String, String] {
```

Comment:
I would like to leave that to represent the issue described in the jira issue. I have already updated this to use AggData